release_notes/v0.8.md

```diff
@@ -0,0 +1,303 @@
+
+- Others
+    - Enhancement the e2e test for GenAIInfra for fixing some bugs([602af5](https://github.com/opea-project/GenAIInfra/commit/602af53742900630a34a4eed9f37980483aa21b3))
+    - Fix bugs for router on handling response from pipeline microservices([ef47f9](https://github.com/opea-project/GenAIInfra/commit/ef47f9db525c16b54d493549b8372946988fce2a))
+    - Improve the examples of codegen and codetrans e2e test([07494c](https://github.com/opea-project/GenAIInfra/commit/07494c0e6ba09030cc8ea464ef783c983b9d5cf7))
+    - Remove the dependencies of common microservices([f6dd87](https://github.com/opea-project/GenAIInfra/commit/f6dd87baf8d569db519e69661ae0d2cdd466fa69))
+    - Add scripts for KubeRay and Ray Cluster([7d3d13](https://github.com/opea-project/GenAIInfra/commit/7d3d13f51f2cfed7be1e92f13f12ef2ff478e1f7))
+    - Enable CI for common components([9e27a0](https://github.com/opea-project/GenAIInfra/commit/9e27a0d424cb3eacbf2cde636426e644ae739212))
+    - Disable common component test([e1cd50](https://github.com/opea-project/GenAIInfra/commit/e1cd50269eebc010bd5f5043a1b4bc8c62a53231))
+    - CI for common: avoid false error in helm test result([876b7a](https://github.com/opea-project/GenAIInfra/commit/876b7a4142e2e1e7a25f25ac279f043c844f1687))
+    - Add the init input for pipeline to keep the parameter information([e25a1f](https://github.com/opea-project/GenAIInfra/commit/e25a1f86e85c452243aacf90a67e47777caf4703))
+    - Adjust CI gaudi version([d75d8f](https://github.com/opea-project/GenAIInfra/commit/d75d8f2e1c356ca26fa09a2e9911de3aff87aa27))
+    - Fix CHART_MOUNT and HFTOKEN for CI([10b908](https://github.com/opea-project/GenAIInfra/commit/10b908abf3b728c9652302efcb071bdc7f8e6426))
+    - Change tgi tag because gaudi driver is upgraded to 1.16.1 ([6796ef](https://github.com/opea-project/GenAIInfra/commit/6796ef2560645c59cdf7b09af9a2d8aa0cb0d5a5))
+    - Update README for new manifests([ec32bf](https://github.com/opea-project/GenAIInfra/commit/ec32bf04459fdbb4c8f99ebd1bac3216ad2e5283))
+    - Support multiple router service in one namespace([0ac732](https://github.com/opea-project/GenAIInfra/commit/0ac73213b501fb5949a5ac8bf7f52d5a4acef580)) 
+    - Improve workflow trigger conditions to be more precise([ab5c8d](https://github.com/opea-project/GenAIInfra/commit/ab5c8d8c07d8f8353315b7ebaf1eb745bf7b28e5))
+    - Remove unnecessary component DocSumGaudi which would cause error([9b973a](https://github.com/opea-project/GenAIInfra/commit/9b973aceb25c307f2c7692c9364ebac9040b9a5d))
+    - Remove chart_test scripts and add script to dump pod status([88caf0](https://github.com/opea-project/GenAIInfra/commit/88caf0df947866ffe609cf60805282970f887429))
+</details>
+
+## Thanks to these contributors
+This release includes valuable contributions from the Intel core team.. We would also like to thank everyone who reported issues or provided feedback. 
+
+Contributors
```

Comment:
Is this a list of contributors across all the repos? I'm not sure we need this full list of 70 contributors, maybe just the top 10 (if any)?

Reply:
I think it's ok to put the contributors here with the small icons.